lib/bottom_navbar_with_indicator.dart

```diff
@@ -95,25 +101,26 @@ class CustomLineIndicatorBottomNavbar extends StatelessWidget {
 /// Documentation
 ///
 /// custom bottom bar items model.
-class CustomBottomBarItems {
+class CustomBottomBarItems<T> {
   /// pass icon with type IconData
-  final IconData icon;
+  final T icon;
```

Comment:
I think it would be better to just allow a Widget, if people want to use IconData they can use an Icon Widget

Reply:
Thank you for your support